doc/README/Tactic/Cong.agda

```diff
@@ -68,6 +68,29 @@ succinct-example m n eq =
 -- to deduce where to generalize. When presented with two sides
 -- of an equality like 'm + n ≡ n + m', it will anti-unify to
 -- 'ϕ + ϕ', which is too specific.
+--
```

Comment:
Can we change the heading of this section from `Limitations` to `Explicit marking` or something like that?

Reply:
Yes. We can and we should. Done.